include/zephyr/bluetooth/services/ots.h

```diff
@@ -250,6 +250,15 @@ enum {
 	BT_OTS_OACP_FEAT_ABORT      = 9,
 };
 
+/*
+ * @enum bt_ots_oacp_write_mode
+ * @brief Mode Parameter for OACP Write Op Code.
+ */
+enum bt_ots_oacp_write_mode {
+	BT_OTS_OACP_WRITE_OP_MODE_NO_TRUNCATE = 0,
+	BT_OTS_OACP_WRITE_OP_MODE_TRUNCATE = 2,
```

Comment:
What is the idea of going from 0 to 2? 

If it is not a bitfield, we can just omit the direct values

Reply:
I see. 

We could consider doing 
```c
BT_OTS_OACP_WRITE_OP_MODE_NONE = 0,
BT_OTS_OACP_WRITE_OP_MODE_TRUNCATE = 2,
```

So that it may be easier to expand on later with non-truncate related modes if they are added to the spec then :)

Reply:
Revisied.